renv/.gitignore

```diff
@@ -5,3 +5,4 @@ local/
 lock/
 python/
 staging/
+profile
```

Comment:
We _don't_ want to commit the profile setting file OR include it in the Docker build, else the container will use the reporting profile.